pajbot/apiwrappers/twitch/helix.py

```diff
@@ -795,3 +795,34 @@ def unban_user(
             {"broadcaster_id": broadcaster_id, "moderator_id": bot_id, "user_id": user_id},
             authorization=authorization,
         )
+
+    def _get_banned_users(
+        self,
+        broadcaster_id: str,
+        authorization,
+        user_id: Optional[str] = None,
+        after_pagination_cursor: Optional[str] = None,
+    ):
+        """Calls the Get Banned Users Helix endpoint using the broadcaster_id & user_id parameter.
+        broadcaster_id is a required field and must match the user ID in authorization."""
+        response = self.get(
+            "/moderation/banned",
+            {
+                "broadcaster_id": broadcaster_id,
+                "user_id": user_id,
+                "first": 100,
+                **self._with_pagination(after_pagination_cursor),
+            },
+            authorization=authorization,
+        )
+
+        pagination_cursor = response["pagination"].get("cursor", None)
+
+        return response["data"], pagination_cursor
+
+    def get_single_banned_user(self, broadcaster_id: str, authorization, user_id: str) -> str:
```

Comment:
Could this return the full information about the banned user instead of just one of the fields?

Reply:
![image](https://user-images.githubusercontent.com/12804673/201897822-cbb84d96-6a90-4ce7-a85e-dfc41dce9ee7.png)